speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -272,16 +460,10 @@ void* ResearcherMenuInit(void* p_nextMenu)
     GfuiAddKey(s_scrHandle, GFUIK_F1, "Help", s_scrHandle, GfuiHelpScreen, NULL);
     GfuiAddKey(s_scrHandle, GFUIK_F12, "Screen-Shot", NULL, GfuiScreenShot, NULL);
 
-    // Set standard Task
-    m_task = TASK_NO_TASK;
-
-    // Set standard interventionType
-    m_interventionType = INTERVENTION_TYPE_NO_SIGNALS;
-
     // Set standard max time
-    char buf[32];
-    sprintf(buf, "%d", m_maxTime);
-    GfuiEditboxSetString(s_scrHandle, m_maxTimeControl, buf);
+    char buf2[32];
+    sprintf(buf2, "%d", m_maxTime);
+    GfuiEditboxSetString(s_scrHandle, m_maxTimeControl, buf2);
```

Comment:
You can re-use the existing buffer
```suggestion
    sprintf(buf, "%d", m_maxTime);
    GfuiEditboxSetString(s_scrHandle, m_maxTimeControl, buf);
```